build/build-image/cross/VERSION

```diff
@@ -1 +1 @@
-v1.13.8-1
+v1.13.9-2
```

Comment:
this remains but the rest of the image is gone?

Reply:
@liggitt -- Yep. There are still a [few things that depend on it](https://cs.k8s.io/?q=cross%2FVERSION&i=nope&files=&repos=) being in that location.

So basically:
- production version: https://github.com/kubernetes/kubernetes/blob/master/build/build-image/cross/VERSION
- prerelease version: https://github.com/kubernetes/release/blob/master/images/build/cross/VERSION

I didn't want to introduce changing the references to this PR, but I am thinking about the best way to do it for a follow-up.

Reply:
ok, as long as its on your radar